htdocs/class/xoopsform/formradio.php

```diff
@@ -180,11 +180,20 @@ public function render()
                 }
                 $ret .= '<td>';
             }
+            
+            if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+                $ret .= '<div class="checkbox-inline"><label>';
+            }
             $ret .= '<input type="radio" name="' . $ele_name . '" id="' . $ele_name . $id_ele . '" title = "' . htmlspecialchars($ele_title, ENT_QUOTES) . '" value="' . htmlspecialchars($value, ENT_QUOTES) . '"';
             if (isset($ele_value) && $value == $ele_value) {
                 $ret .= ' checked';
             }
-            $ret .= $ele_extra . ' />' . "<label name='xolb_{$ele_name}' for='" . $ele_name . $id_ele . "'>" . $name . '</label>' . $ele_delimeter;
+            if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+                $ret .= $ele_extra . ' /> ' . $name . $ele_delimeter;
+                $ret .= '<label></div>';
+            } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->